app/assets/javascripts/host_edit.js

```diff
@@ -289,7 +292,10 @@ function os_selected(element){
     data: attrs,
     type:'post',
     url: url,
-    complete: function(){  $(element).indicator_hide();},
+    complete: function(){
```

Comment:
i wonder if there is no event you can simply tap into instead of changing every ajax call